From e9e335352a0815018d7db98c165d3669c3d8804f Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@gmail.com>
Date: Tue, 23 Jan 2024 09:56:24 -0500
Subject: [PATCH] aufs: update remove_page to remove_folio

Commit af7628d6ec19 [fs: convert error_remove_page to
error_remove_folio]
switches remove_page to the folio equivalent. We switch
the aufs definitions to match.

Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 fs/aufs/dynop.c | 2 +-
 fs/aufs/file.c  | 6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/fs/aufs/dynop.c b/fs/aufs/dynop.c
index 54fd16ecb6d3..f62b3b56291d 100644
--- a/fs/aufs/dynop.c
+++ b/fs/aufs/dynop.c
@@ -192,7 +192,7 @@ static void dy_aop(struct au_dykey *key, const void *h_op,
 	DySetAop(launder_folio);
 	DySetAop(is_partially_uptodate);
 	DySetAop(is_dirty_writeback);
-	DySetAop(error_remove_page);
+	DySetAop(error_remove_folio);
 	DySetAop(swap_activate);
 	DySetAop(swap_deactivate);
 	DySetAop(swap_rw);
diff --git a/fs/aufs/file.c b/fs/aufs/file.c
index 76ad444328db..2e3bdac4699a 100644
--- a/fs/aufs/file.c
+++ b/fs/aufs/file.c
@@ -827,8 +827,8 @@ static bool aufs_is_partially_uptodate(struct folio *folio, size_t from,
 static void aufs_is_dirty_writeback(struct folio *folio, bool *dirty,
 				    bool *writeback)
 { AuUnsupport(); }
-static int aufs_error_remove_page(struct address_space *mapping,
-				  struct page *page)
+static int aufs_error_remove_folio(struct address_space *mapping,
+				  struct folio *folio)
 { AuUnsupport(); return 0; }
 static int aufs_swap_activate(struct swap_info_struct *sis, struct file *file,
 			      sector_t *span)
@@ -857,7 +857,7 @@ const struct address_space_operations aufs_aop = {
 	.launder_folio		= aufs_launder_folio,
 	.is_partially_uptodate	= aufs_is_partially_uptodate,
 	.is_dirty_writeback	= aufs_is_dirty_writeback,
-	.error_remove_page	= aufs_error_remove_page,
+	.error_remove_folio	= aufs_error_remove_folio,
 	.swap_activate		= aufs_swap_activate,
 	.swap_deactivate	= aufs_swap_deactivate,
 	.swap_rw		= aufs_swap_rw
-- 
2.39.2

